database/migrations/model/Version20240319123214.php

```diff
@@ -0,0 +1,55 @@
+<?php namespace Database\Migrations\Model;
+/**
+ * Copyright 2024 OpenStack Foundation
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+use Doctrine\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema as Schema;
+use LaravelDoctrine\Migrations\Schema\Table;
+/**
+ * Class Version20240319123214
+ * @package Database\Migrations\Model
+ */
+final class Version20240319123214 extends AbstractMigration
+{
+    use CreateTableTrait;
+
+    const TableName = 'SummitLeadReportSetting';
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema): void
+    {
+        self::createTable($schema, self::TableName, function(Table $table){
+            $table->string("Columns")->setNotnull(true);
```

Comment:
spec saids use json type
not string
https://dev.mysql.com/doc/refman/8.0/en/json.html
https://www.digitalocean.com/community/tutorials/working-with-json-in-mysql